### ISSUE TITLE
AcquireTokenSilentAsync using Integrated authentication on Windows (Kerberos) : password_required_for_managed_user

### ISSUE THREAD

**Issue by MartinKuhne** [AzureAD/azure-activedirectory-library-for-dotnet#1094]
I tried out this code
AcquireTokenSilentAsync using Integrated authentication on Windows (Kerberos)

Unhandled Exception: System.AggregateException: One or more errors occurred. ---> Microsoft.IdentityModel.Clients.ActiveDirectory.AdalException: password_required_for_managed_user: Password is required for managed user
   at Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Flows.AcquireTokenNonInteractiveHandler.<PreTokenRequestAsync>d__5.MoveNext()

```
            // use an auth context that's tenant agnostic
            var authenticationContext = new AuthenticationContext("https://login.microsoftonline.com/common");
            var userCredential = new UserCredential();

            try
            {
                var result = await authenticationContext.AcquireTokenAsync("https://graph.microsoft.com/.default", "https://tenant.onmicrosoft.com/someguid", userCredential);
                return result.UserInfo.UniqueId;
            }
```

cf. https://stackoverflow.com/questions/50919092/get-error-password-is-required-for-managed-user-when-authenticating-user-with

which contradicts this article stating Kerberos will not work for Azure AD (which sounds plausible)

**Comment by henrik-me** [comment]
@MartinKuhne : Not sure I fully understand what you are trying to achieve. Is the purpose of this issue to tell us that the error message might be wrong or do you have other expectations?
@jmprieur : Shouldn't Kerberos work with seamless SSO?

**Comment by henrik-me** [comment]
#1098

**Comment by jmprieur** [comment]
@henrik-me : it should. I've involved the seamless SSO team.

**Comment by jmprieur** [comment]
@bgavrilMS FYI

**Comment by bgavrilMS** [comment]
It looks to be the same issue as https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1398 

You get this strange exception for a "managed" user when doing an Integrated Windows Auth (IWA) flow. Managed users are those created in AAD without an AD backing. IWA is based on an old protocol that works by calling some apis on the AD server. Since there is no AD server, we can't enable this flow.

I've updated the error messages (PR pending) and added details to the docs. Proposing to close this.
